docs/source/installation.rst

```diff
@@ -332,25 +336,25 @@ We use `uwsgi`_ to interface python and nginx::
   cp python_plugin.so ~/envs/mozdef/bin/
   cp uwsgi ~/envs/mozdef/bin/
 
-  cp -r ~/MozDef/rest   ~/envs/mozdef/
-  cp -r ~/MozDef/loginput   ~/envs/mozdef/
-  mkdir ~/envs/mozdef/logs
+  cd ..
+  cp -r ~/mozdef/rest   ~/envs/mozdef/
+  cp -r ~/mozdef/loginput   ~/envs/mozdef/
 
   cd ~/envs/mozdef/rest
   # modify config file
   vim index.conf
-  # modify uwsgi.ini
-  vim uwsgi.ini
-  uwsgi --ini uwsgi.ini
+  # modify restapi.ini
+  vim restapi.ini
```

Comment:
I've pointed to the systemd unit files to start these services.
I'll put additional information on how to start them manually, but I think most will want to use a script in order to be able to have a better time at stopping the process rather than digging through documentation to find out how to do it in uwsgi.